observability/distr_tracing/distr-tracing-tempo-updating.adoc

```diff
@@ -1,11 +1,16 @@
 :_mod-docs-content-type: ASSEMBLY
-[id="dist-tracing-tempo-updating"]
+[id="distr-tracing-tempo-updating"]
 = Upgrading
 include::_attributes/common-attributes.adoc[]
-:context: dist-tracing-tempo-updating
+:context: distr-tracing-tempo-updating
 
 toc::[]
 
+[TIP]
+====
+For upgrading the deprecated {JaegerShortName}, see xref:../../observability/distr_tracing/distr_tracing_jaeger/distr-tracing-jaeger-updating.adoc#dist-tracing-jaeger-updating[Upgrading] in the {JaegerShortName} documentation.
```

Comment:
```suggestion
For information about upgrading the deprecated {JaegerShortName}, see xref:../../observability/distr_tracing/distr_tracing_jaeger/distr-tracing-jaeger-updating.adoc#dist-tracing-jaeger-updating[Upgrading] in the {JaegerShortName} documentation.
```

Reply:
Thank you so much for this suggestion. Fixed.